specification.markdown

```diff
@@ -778,3 +778,14 @@ minOccurs: 0
 type: Binary
 
 description: Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+# Considerations for Reading EBML Data
+
+The following scenarios describe events when reading `EBML Documents` and the REQUIRED behavior by an `EBML Reader`.
+
+If a `Master Element` contains a `CRC-32 Element` that doesn't validate, then the `EBML Reader` MUST ignore all contained data except for `Descendant Elements` which contain their own valid `CRC-32 Element`.
```

Comment:
That's only for systems that don't want to handle damaged data. A player (let's call it VLC) will try to read the data anyway and extract as much as it finds valid. You cannot say that players should ignore a Cluster because of single byte (P2P unfinished download for example) is missing.